main.tf

```diff
@@ -1,48 +1,34 @@
 module "cpu_utilization_high_alarm_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.17.0"
-  enabled    = var.enabled
-  name       = var.name
-  namespace  = var.namespace
-  stage      = var.stage
-  attributes = compact(concat(var.attributes, ["cpu", "utilization", "high"]))
-  delimiter  = var.delimiter
-  tags       = var.tags
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.21.0"
+  attributes = ["cpu", "utilization", "high"]
+
+  context = module.this.context
 }
 
 module "cpu_utilization_low_alarm_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.17.0"
-  enabled    = var.enabled
-  name       = var.name
-  namespace  = var.namespace
-  stage      = var.stage
-  attributes = compact(concat(var.attributes, ["cpu", "utilization", "low"]))
-  delimiter  = var.delimiter
-  tags       = var.tags
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.21.0"
+  attributes = ["cpu", "utilization", "low"]
+
+  context = module.this.context
 }
 
 module "memory_utilization_high_alarm_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.17.0"
-  enabled    = var.enabled
-  name       = var.name
-  namespace  = var.namespace
-  stage      = var.stage
-  attributes = compact(concat(var.attributes, ["memory", "utilization", "high"]))
-  delimiter  = var.delimiter
-  tags       = var.tags
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.21.0"
+  attributes = ["memory", "utilization", "high"]
+
+  context = module.this.context
 }
 
 module "memory_utilization_low_alarm_label" {
-  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.17.0"
-  enabled    = var.enabled
-  name       = var.name
-  namespace  = var.namespace
-  stage      = var.stage
-  attributes = compact(concat(var.attributes, ["memory", "utilization", "low"]))
-  delimiter  = var.delimiter
-  tags       = var.tags
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.21.0"
+  attributes = ["memory", "utilization", "low"]
+
+  context = module.this.context
 }
 
 locals {
+  enabled = module.this.enabled
```

Comment:
@okgolove I'm thinking we just use module.this.enabled instead, so it matches all the other modules. This may be a case of premature optimisation. Easy to change later if required.